programs/fileio.c

```diff
@@ -759,9 +768,16 @@ typedef struct {
     ZSTD_CStream* cctx;
 } cRess_t;
 
+static unsigned int FIO_log2(const size_t x)
```

Comment:
note : `fileio.c` already defines `FIO_highbit64()`, which does the same,
except that input type is `unsigned long long` instead of `size_t`.

Recommendation : either use existing one,
or create another one due to input type difference, but ensure naming consistency.

Reply:
Ah okay, didn't know about that function. I'll just use that one.